### PR TITLE
Add code sniffer/fixer targets to make

### LINKED ISSUE #24: Add phpcs checks for custom code
We have no sniffer check for custom code
Let's add proper section to Makefile

```
phpcs:
	docker run --rm -v $(shell pwd)/src/$(PROFILE_NAME):/work skilldlabs/docker-phpcs-drupal phpcs --standard=Drupal,DrupalPractice --extensions=php,module,inc,install,profile,theme .

phpcbf:
	docker run --rm -v $(shell pwd)/src/$(PROFILE_NAME):/work skilldlabs/docker-phpcs-drupal phpcbf --standard=Drupal,DrupalPractice --extensions=php,module,inc,install,profile,theme .

```

### PR DESCRIPTION
**Pull request by andypost**
Now backenders can use `make phpcs` & `make phpcbf`

Closes #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,3 +56,5 @@ For Linux install <a href="https://docs.docker.com/compose/install/" target="_bl
 * `make info` - Show project info (IP).
 * `make chown` - Change permissions inside container. Use it in case you can not access files in _build_. folder from your machine.
 * `make exec` - docker exec into php container.
+* `make phpcs` - check codebase with `phpcs` shiffers to make sure it conforms https://www.drupal.org/docs/develop/standards.
+* `make phpcbf` - fix codebase accordind Drupal standards https://www.drupal.org/docs/develop/standards.
```
Comment (jbguerraz) [inline README.md]: "accordind" => "according to"

File: Makefile [hunk 2]
```diff
@@ -75,3 +75,11 @@ front:
 
 iprange:
 	$(shell grep -q -F 'IPRANGE=' .env || echo "\nIPRANGE=$(shell docker network inspect $(COMPOSE_PROJECT_NAME)_front --format '{{(index .IPAM.Config 0).Subnet}}')" >> .env)
+
+phpcs:
+	docker run --rm -v $(shell pwd)/src/$(PROFILE_NAME):/work skilldlabs/docker-phpcs-drupal phpcs --standard=Drupal --extensions=php,module,inc,install,test,profile,theme,info .
+	docker run --rm -v $(shell pwd)/src/$(PROFILE_NAME):/work skilldlabs/docker-phpcs-drupal phpcs --standard=DrupalPractice --extensions=php,module,inc,install,test,profile,theme,info .
+
```
Comment (tkyryliuk) [inline Makefile]: According to documentation we can specify standards separated by commas: --standard=Drupal,DrupalPractice

File: Makefile [hunk 3]
```diff
@@ -75,3 +75,11 @@ front:
...
+phpcbf:
+	docker run --rm -v $(shell pwd)/src/$(PROFILE_NAME):/work skilldlabs/docker-phpcs-drupal phpcbf --standard=Drupal --extensions=php,module,inc,install,test,profile,theme,info .
+	docker run --rm -v $(shell pwd)/src/$(PROFILE_NAME):/work skilldlabs/docker-phpcs-drupal phpcbf --standard=DrupalPractice --extensions=php,module,inc,install,test,profile,theme,info .
```
Comment (tkyryliuk) [inline Makefile]: Same here we can use --standard=Drupal,DrupalPractice and on my project this command doesn't work without --no-patch flag.

**Comment by andypost** [comment]
I prefer to keep each standard separate to allow manual fixing/reusing/disabling them

btw it need reroll(

**Comment by andypost** [comment]
rebased/merged

File: Makefile (hunk 3 above)
Comment (andypost) [inline Makefile]: I added `patch` package into php docker image

**Pull request merged**